docs/src/admin-manual/configuration.md

```diff
@@ -832,14 +826,13 @@ zipPIP = false
   [preservation engine], Enduro will then regularly poll the preservation engine
   for updates. This setting determines how frequently Enduro will poll for
   updates, with the default value being 10 seconds (`10s`). Interval values
-  should be compatible with what the [ParseDuration] function's values of the
-  GoLang `time` package - valid time units are "ns", "us" (or "µs"), "ms", "s",
-  "m", and "h".
+  should be compatible with the [GoLang] [ParseDuration] function — valid values
```

Comment:
Sorry, one more suggested change "Interval values *must* be compatible with the [GoLang] [ParseDuration] function"